reps/2022-08-31-actor-affinity-apis.md

```diff
@@ -0,0 +1,538 @@
+
+## Summary
+### General Motivation
+
+Provides a set of lightweight actor affinity and anti-affinity scheduling interfaces.
+Replacing the heavy PG interface to implement collocate or spread actors.
```

Comment:
To clarify, this proposal only allows scheduling of actors relative to other actors, correct?

Reply:
Yes. The current implementation of ActorAffinity is to choose affinity or anti-affinity to other actors when scheduling actors. However, it can also be extended to the relationship between task and actor in the future. 
![image](https://user-images.githubusercontent.com/11072802/210948806-b89bd4ec-cd1b-4441-ae7c-eb431603b38f.png)